sql/moz-fx-data-shared-prod/search_derived/search_clients_daily_v8/query.sql

```diff
@@ -46,77 +46,9 @@ CREATE TEMP FUNCTION organicize_source_or_type(engine STRING, original STRING) A
   END
 );
 
-WITH overactive AS (
```

Comment:
Overactive filter is identical to the one in `clients_daily` so it's not needed anymore